internal/blobstore/blobstore.go

```diff
@@ -292,8 +292,9 @@ func NewRefs(n int) *Refs {
 // Add records that the given hash is referenced.
 // It ignores the hash if it's invalid.
 func (r *Refs) Add(hash string) {
-	data, ok := decodeHash(hash)
-	if !ok {
+	data, err := decodeHash(hash)
+	if err != nil {
+		logger.Errorf("cannot add bad hash %q: %v", hash, err)
```

Comment:
This really shouldn't be an error I don't think. We use this with refs that are empty a lot prev5blobhashes that aren't specified for instance.